lib/src/rate_limit.dart

```diff
@@ -224,15 +224,60 @@ extension RateLimit<T> on Stream<T> {
   /// Returns a Stream  which collects values and emits when it sees a value on
```

Comment:
Stream -> \[Stream\]  (or just "stream" as a word instead of a source name).

Double space after "Stream  "

Should use the word "buffer" in the first line.